infrastructure/lambda/template.yaml

```diff
@@ -757,7 +697,7 @@ Resources:
     Properties:
       Name: !Sub "/${AWS::StackName}/verifiableCredentialKmsSigningKeyId"
       Type: String
-      Value: !FindInMap [ VerifiableCredentialKmsSigningKeyMapping, Environment, !Ref 'Environment' ]
+      Value: !Ref AddressCriVcSigningKey
```

Comment:
no need to refer to a mapping, `!Ref AddressCriVcSigningKey` gives the KMS key id